bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -2050,6 +2059,53 @@ protected void readLedgerIndexEntries(long ledgerId) throws IOException {
     }
 
     /**
+     * Read ledger index entries
+     * 
+     * @param ledgerId
+     * @return
+     * @throws IOException if failed to init KeyValueStorageRocksDB
+     */
+	protected int readLedgerIndexEntriesFromDbLedgerStorage(long ledgerId) throws IOException {
```

Comment:
Sure. I was thinking more like a static method in there or to add a class in the `..storage.ldb.` package, just to keep all the `DbLederStorage` specific code contained in one place as much as possible.